apps/react-native/package.json

```diff
@@ -14,6 +14,7 @@
   },
   "dependencies": {
     "@brandingbrand/fsapp": "^11.77.1",
+    "fsdev": "file:../../../local_npm_modules/fsdev",
```

Comment:
I understand this is still WIP - just noting to migrate to `@brandingbrand/kernel-component-dev`

Reply:
Removed